### ISSUE TITLE
Add option to `no-unassigned-import` for allowed glob

### ISSUE THREAD

**Issue by kentcdodds** [benmosher/eslint-plugin-import#671]
When using the style and css webpack loaders, sometimes I want to be able to import a CSS file which will result in injecting the styles in the DOM (a side-effect) but this is acceptable in some use cases (I realize there are CSS modules, but importing things like bootstrap etc.)

So I suggest:

```javascript
"import/no-unassigned-import": [2, {
  "allowed": ["*.css"]
}]
```

Thoughts?

**Comment by jfmengels** [comment]
Sounds good to me :+1:

**Comment by kentcdodds** [comment]
I'd be happy to. I'll see when I can find the time. I'll add a comment in here when I've started working on it... Anyone else can feel free before me though :)

**Comment by jfmengels** [comment]
That'd be awesome! I'm pretty sure similar work has been made in no-extraneous-dependencies (at least regarding globs) FYI.
